sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace_keys.py

```diff
@@ -4,7 +4,7 @@
 
 from typing import List, Optional
 
-from azure.ai.ml._restclient.v2022_12_01_preview.models import ListWorkspaceKeysResult
+from azure.ai.ml._restclient.v2023_02_01_preview.models import ListWorkspaceKeysResult
```

Comment:
According to previous comment, can workspace continue to use 2022_12_01_preview version?